xbatcher/loaders/keras.py

```diff
@@ -49,7 +49,7 @@ def __getitem__(self, idx: int) -> Tuple[Any, Any]:
             xr.concat(
                 (
                     self.X_generator[idx][key]
-                    for key in list(self.X_generator[idx].keys())
+                    for key in list(self.X_generator.input_dims)
```

Comment:
Not quite sure what the intention of this concatenation is for, and I couldn't quite understand it looking at `test_keras_loaders.py` either. Please double check.

Reply:
It was a rough way to prototype the dataset -> dataarray conversion. Since this it a very new, unreleased prototype, one option is to simplify to data loader to only expect data array inputs and later generalize for dataset or dataarrays.

Reply:
Ah ok, I see that this is only 2 days old and added in #73. We could definitely walk back on this and support DataArray only for now. Let me refactor this to be `xr.DataArray` specific only.

Reply:
sounds great, thanks @weiji14!